warehouse/query-core/src/test/java/datawave/query/jexl/functions/ContentFunctionQueryTest.java

```diff
@@ -0,0 +1,476 @@
+    
+    private static final String PASSWORD = "";
+    private static final String AUTHS = "ALL";
+    
+    private static final String formatPattern = "yyyyMMdd HHmmss.SSS";
+    private static final SimpleDateFormat formatter = new SimpleDateFormat(formatPattern);
+    
+    private static final String BEGIN_DATE = "20000101 000000.000";
+    private static final String END_DATE = "20020101 000000.000";
+    
+    private static final String USER = "testcorp";
+    private static final String USER_DN = "cn=test.testcorp.com, ou=datawave, ou=development, o=testcorp, c=us";
+    
+    private static final Configuration conf = new Configuration();
+    private static final String TEST_DATA = "datawave/query/jexl/functions/ContentFunctionQueryExample.csv";
+    
+    @Inject
+    @SpringBean(name = "EventQuery")
+    ShardQueryLogic logic;
+    
+    private static InMemoryInstance instance;
+    
+    @Deployment
+    public static JavaArchive createDeployment() throws Exception {
+        return ShrinkWrap
+                        .create(JavaArchive.class)
+                        .addPackages(true, "org.apache.deltaspike", "io.astefanutti.metrics.cdi", "datawave.query", "datawave.webservice.query.result.event")
+                        .addClass(TestDatawaveEdgeDictionaryImpl.class)
+                        .deleteClass(datawave.query.metrics.QueryMetricQueryLogic.class)
+                        .deleteClass(datawave.query.metrics.ShardTableQueryMetricHandler.class)
+                        .addAsManifestResource(
+                                        new StringAsset("<alternatives>" + "<stereotype>datawave.query.tables.edge.MockAlternative</stereotype>"
+                                                        + "</alternatives>"), "beans.xml");
+    }
+    
+    @BeforeClass
+    public static void setupClass() throws Exception {
+        System.setProperty("subject.dn.pattern", "(?:^|,)\\s*OU\\s*=\\s*My Department\\s*(?:,|$)");
+        
+        byte[] data = IOUtils.toByteArray(ClassLoader.getSystemResource(TEST_DATA).openStream());
+        
+        setupConfiguration(conf);
+        
+        TestContentIndexingHandler dataTypeHandler = new TestContentIndexingHandler();
+        dataTypeHandler.setup(new TaskAttemptContextImpl(conf, new TaskAttemptID()));
+        
+        TestIngestHelper ingestHelper = new TestIngestHelper();
+        ingestHelper.setup(conf);
+        
+        // create and process events with test data
+        RawRecordContainer record = new RawRecordContainerImpl();
+        Multimap<BulkIngestKey,Value> keyValues = HashMultimap.create();
+        
+        record.clear();
+        record.setDataType(new Type(DATA_TYPE_NAME, TestIngestHelper.class, null, null, 1, null));
+        record.setRawFileName("example.dat");
+        record.setRawRecordNumber(1);
+        record.setDate(formatter.parse(BEGIN_DATE).getTime());
+        record.setRawDataAndGenerateId(data);
```

Comment:
This method seems to have been refactored into two calls.
record.setRawData(data);
record.generateId(null);